docs/Interoperator-authentication.md

```diff
@@ -0,0 +1,52 @@
+# Service Fabrik Inter-operator Authentication
+
+##  Abstract
+
+This document describes the basic and mTLS authentication supported by the Service Fabrik inter-operator broker.
+
+## Table of Content
+- [Service Fabrik Inter-operator Basic Architecture](#service-fabrik-inter-operator-basic-architecture)
+  - [Abstract](#abstract)
+  - [Table of Content](#table-of-content)
+  - [Context](#context)
+  - [BasicAuthentication](#basic-authentication)
+  - [mTLSAuthentication](#out-of-the-box-mtls-authentication)
+    - [CertificateIdentityVerification](#certificate-identity-verification)
+
+## Context
+The following two authentication mechanisms are supported for communicating with the brokers:
+  - Basic Authentication: validates the username and password.
+  - mTLS Authentication: validates the client certificate's subject pattern.
+
+## Basic Authentication
+To use Basic Authentication, the username and password should be provided during the registration of the broker.
+This is relevant for both subaccount-scoped registration done by smctl, and global registration done by the Cross Registration Service (xRS) API.
```

Comment:
```suggestion
```
The subaccount-scoped registration and the Cross Registration Service (xRS) API is specific to SAP. I feel we can skip this sentence.